src/lib/components/forms/form/form-components/select/select-box.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-form-field
+    class="w-100 form-input"
```

Comment:
```suggestion
    class="w-100"
```
Die Klasse gibt es nicht.